Libplanet/Tx/InvalidActionTypeException.cs

```diff
@@ -0,0 +1,32 @@
+using System;
+using System.Runtime.Serialization;
+
+namespace Libplanet.Tx
+{
+    [Serializable]
+    public class InvalidActionTypeException : Exception
```

Comment:
Could we seal this exception? (or, should we open to inherit?)

Reply:
Sure, it doesn't need to be inherited.